daffodil-test-ibm1/src/test/resources/test-suite/tresys-contributed/AU.tdml

```diff
@@ -7,23 +7,23 @@
     description="Binary values packed and bsd">
     <document>
       <documentPart type="byte"><![CDATA[
-    1700 2300 2700 4119 8c01 8b01 1c72 1b72 7542 636e 2068 666f 4e20 6d75 6562 7372
+    0017 0023 0027 1941 018c 018b 721c 721b 4275 6e63 6820 6f66 204e 756d 6265 7273
     ]]></documentPart>
     </document>
     <infoset>
       <dfdlInfoset>
         <list xmlns="http://www.example.org/example1" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-          <bcd type="xsd:int">0017</bcd>
-          <bcd type="xsd:int">0023</bcd>
-          <bcd type="xsd:int">0027</bcd>
+          <bcd type="xsd:int">17</bcd>
+          <bcd type="xsd:int">23</bcd>
+          <bcd type="xsd:int">27</bcd>

```

Comment:
Should the ``type`` here be xsd:unsignedInt to match your changes above. Daffodil currently ignores this attribute, but we might check it one day.

Reply:
I believe we should remove these type="xsd:int". We're not generating them, not checking them, they're just misleading at best currently.